manifests/rhsm_reconfigure_script.pp

```diff
@@ -8,13 +8,13 @@
 
   concat::fragment { "${title}+script_start":
     target  => $title,
-    content => "#!/bin/bash\n\n",
+    content => "#!/bin/bash\n\nset -e\n",
     order   => '01',
   }
 
   concat::fragment { "${title}+default_ca_data":
     target  => $title,
-    content => "read -r -d '' KATELLO_DEFAULT_CA_DATA << EOM\n",
+    content => "read -r -d '' KATELLO_DEFAULT_CA_DATA << EOM || true\n",
```

Comment:
In which case does this fail?

Reply:
I think, every time. Do not know why, though
```
root@steve-ratzloff:~# read -d '' -r TEST << EOF
> asdf
> EOF
root@steve-ratzloff:~# echo $TEST $?
asdf 1
```

Reply:
You're right: from ```man read```: 
> The exit status is zero, unless end-of-file is encountered

Reply:
Thanks for diving into this.
But now i remember, that is why `while read LINE` works.